notebook-tiny/pyspark.ipynb

```diff
@@ -425,7 +425,7 @@
     "import json\n",
     "\n",
     "# define the fields we want to keep\n",
-    "interesting_fields = ['agent', 'author', 'copr', 'user', 'msg', 'meeting_topic', 'name', 'owner', 'package']\n",
+    "interesting_fields = ('agent', 'author', 'copr', 'user', 'msg', 'meeting_topic', 'name', 'owner', 'package')\n",
```

Comment:
Do you thing it would be good to have this list as immutable? Or will Apache Spark take care of the closure by itself ?